File: server/Models/Recipe.cs
namespace W9_CK.Models;

public class Recipe : RepoUserContent<int>
{
    public string Title { get; set; }
    public string Instructions { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
}

public class FavoriteRecipe : Recipe
{
    public int FavoriteId { get; set; }
}

File: client/src/models/Recipe.js
export class Recipe {
	constructor(data) {
		this.id = data.id;
		this.category = data.category;

		this.title = data.title;
		this.img = data.img;
		this.instructions = data.instructions;

		this.creatorId = data.creatorId;
		this.creator = data.creator;
		this.favoriteId = data.favoriteId || null;
		this.favorited = Boolean(data.favoriteId);
	}
}
